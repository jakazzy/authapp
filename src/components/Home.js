import React from 'react'
import { Link } from 'react-router-dom';

const Home =()=>  {
return (
    <div>
        <nav className="navbar">
            <ul>
                <li className="link-button"><Link to="/signin">Log In</Link ></li>
                <li className="link-button right"><Link to="/signup">SignUp</Link ></li>
            </ul>
        </nav>
        <section>
            <h3  data-testid="titleText" id="Homepage-text" className="title-text">Customer Success Portal</h3>
        </section>
    </div>   
)

}

export default Home

