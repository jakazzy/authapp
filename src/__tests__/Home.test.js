import React from 'react'
import { BrowserRouter,Route, Link } from 'react-router-dom';
import { render, cleanup, fireEvent } from '@testing-library/react'
import 'jest-dom/extend-expect'
import Home from './../components/Home'

afterEach(cleanup);

describe('renders without crashing', () => {
    test('renders home page', () => {
    const { asFragment} = render(
                                    <BrowserRouter>
                                    <Route path="/" component={ Home} />
                                        {/* <Link to="/"><Home/></Link> */}
                                    </BrowserRouter>);
    expect(asFragment()).toMatchSnapshot();
});
})
