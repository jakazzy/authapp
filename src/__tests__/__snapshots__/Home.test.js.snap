// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing renders home page 1`] = `
<DocumentFragment>
  <div>
    <nav
      class="navbar"
    >
      <ul>
        <li
          class="link-button"
        >
          <a
            href="/signin"
          >
            Log In
          </a>
        </li>
        <li
          class="link-button right"
        >
          <a
            href="/signup"
          >
            SignUp
          </a>
        </li>
      </ul>
    </nav>
    <section>
      <h3
        class="title-text"
        id="Homepage-text"
      >
        Customer Success Portal
      </h3>
    </section>
  </div>
</DocumentFragment>
`;
